client/src/ui/stories/molecules/OrcidDetails.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { OrcidDetails, AuthorDetails } from '../../molecules';
+import '../../../core/styles/index.scss';
+import centered from '@storybook/addon-centered/react';
```

Comment:
I'd like to chat about the styles and folder structure to understand a bit more.